### PR TITLE
SashaBranch

### PR DESCRIPTION
**Pull request by AleksandraGorodnya**
Homework

### CODE REVIEW

File: team-members/sasha-gorodnya.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+       </section>
+  </main>
+     <footer>
+    <h4>Сonnect with me on Instagram:</h4>
+    <button class="btn btn-success"
+   onclick=" window.open('https://www.instagram.com/sandelfini/?hl=uk','_blank')">Instagram</button>
+   <form action="/action_page.php"><br>
+     <fieldset>
+       <legend><strong><section id="part4">Contact me:</section></strong></legend>
+       <form action="/action_page.php" autocomplete="on">
```
Comment (kate-sagittarius) [inline team-members/sasha-gorodnya.html:100]: form shouldn't be inside the other form + this tag has no closing tag

File: team-members/sasha-gorodnya.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>My resume</title>
+    </head>
```
Comment (kate-sagittarius) [inline team-members/sasha-gorodnya.html]: check all file and correct indents

**Review by kate-sagittarius — CHANGES_REQUESTED** [review]
Please, correct all indents + form tag and commit changes and then we will merge it

File: team-members/sasha-gorodnya.html (hunk 1 above)
Comment (AleksandraGorodnya) [reply team-members/sasha-gorodnya.html:100]: done

File: team-members/sasha-gorodnya.html (hunk 2 above)
Comment (AleksandraGorodnya) [reply team-members/sasha-gorodnya.html]: done

**Pull request merged**